### ISSUE TITLE
SHELL_PLUS_PRE_IMPORTS requires a list, but the docs say to use a tuple

### ISSUE THREAD

**Issue by akxlr** [django-extensions/django-extensions#1023]
In my setup (Django 1.10.5), this works:

```
SHELL_PLUS_PRE_IMPORTS = [
    ('myapp.shell_plus', '*')
]
```

while this doesn't work:

```
SHELL_PLUS_PRE_IMPORTS = (
    ('my_app.shell_plus', '*')
)
```

however the docs say to use the latter.

**Comment by rootart** [comment]
Hey, @akxlr I think explanation below could help to your case.

```python
SHELL_PLUS_PRE_IMPORTS = (
    ('my_app.shell_plus', '*')
)
```
will be equal to 

```python
SHELL_PLUS_PRE_IMPORTS = ('my_app.shell_plus', '*')
```
which will be directly translated to 
```python
import my_app.shell_plus
import *
``` 
additing comma after the inner tuple will solve the case

```python

SHELL_PLUS_PRE_IMPORTS = (
    ('my_app.shell_plus', '*'),
)
```

@trbs do you think any patches are required here? And if so will gladly help if you can give a quick direction.

**Comment by trbs** [comment]
@rootart thanks for the explanation and offer to help !

Your absolutely right @akxlr's problem is "normal" Python where a tuple with 1 element requires the additional comma otherwise it's threaded as a grouping expressions and simplified.

@akxlr has a valid point that in the documentation we use a tuple and not a list in the example.
I committed a fix for that in 73b4820dc74c687e50e13ca0419b615932566bcf

Think that should solve any misunderstanding caused by the example right ? (If not feel free to reopen)

Thanks !
